### PR TITLE
[enocean] Improve capability listing to align with EEP documentation

### PR DESCRIPTION
**Pull request by dilyanpalauzov**
This updates the descriptions for EnOcean devices, based on the table on page 132 of https://www.enocean-alliance.org/wp-content/uploads/2017/05/EnOcean_Equipment_Profiles_EEP_v2.6.7_public.pdf, whether the properties from the subject are present.

This makes the device types more distinguishable from each other, and eases the users.

In particular it replaces “dimming” with “configurable dimming” where appropriate.

I have not checked if the class implementation does support these properties (match the table): in case of D2_01_0A I have removed “energy measurement” from the description.

Related to https://github.com/openhab/openhab-addons/pull/17429 and https://github.com/openhab/openhab-addons/pull/17450.

### CODE REVIEW

**Comment by dilyanpalauzov** [comment]
receivingEEPId and sendingEEPId are kept identical.  I do not know if this is correct.

File: bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties
Comment (lovery) [inline bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties:196]:
```suggestion
thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0C = Switching, pilot wire mode, power failure detection, power and energy measurement (EEP: D2_01_0C)
```

**Comment by lovery** [comment]
Can you also sign your commit - this will fix the DCO check?

Do you know if `power failure detection` is implemented?

By the way the `pilot wire mode` for `D2_01_0C` is still not merged because it needs testing - #17450 if you have such a device and you have time to test the implementation I will appreciate it.

**Comment by dilyanpalauzov** [comment]
> Do you know if receivingEEPId and sendingEEPId should be identical or distinct?

I mean their descriptions, as changed here.

**Comment by lovery** [comment]
@dilyanpalauzov I think they are identical everywhere.

File: bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties
Comment (lsiepel) [reply bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties:195]: In this case 'with' extends the failure detection capability. Actually 'power and energy measurement' is a seperate capability and not an extension of the detectoin, hence the suggested `,` is IMO a good improvement.

**Comment by lsiepel** [comment]
Besides the last comment that's waiting for @lovery comment, i wonder if we can merge this or if it needs to wait before the pilot wire PR is merged. As this update states pilot wire capability's

**Comment by dilyanpalauzov** [comment]
This improves the descriptions of the devices, which otherwise had very identical spelling, and were hard to distinguish.   Whether the listed features are supported by OpenHAB is secondary.

File: bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties
Comment (lovery) [reply bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties]: You are right `D2_01_0C` doesn't have Power failure detection (the table is quite difficult to read - I mixed the columns).

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM.

Let's hope we can merge the pilot wire mode PR aslo before 4.3.0

**Pull request merged**
